internal/ctxval/context_values.go

```diff
@@ -56,9 +59,13 @@ func WithRequestNumber(ctx context.Context, num uint64) context.Context {
 	return context.WithValue(ctx, requestNumCtxKey, num)
 }
 
-// Account returns current account model or nil when not set.
+// Account returns current account model or panics when not set
 func AccountId(ctx context.Context) int64 {
-	return ctx.Value(accountIdCtxKey).(int64)
+	value := ctx.Value(accountIdCtxKey)
+	if value == nil {
+		panic(MissingAccountInContextError)
+	}
+	return value.(int64)
 }
```

Comment:
When testing, I realized that if you call a DAO function that requires account in context but it is not present (e.g. worker process did not have it previously), an nil pointer panic occurs.

This improves the error message, it now says: operation requires account_id in context